rmgpy/pdep/sls.py

```diff
@@ -123,6 +123,208 @@ def states_to_configurations(network,indices,state,exclude_association=False):
             xs[i+network.n_isom] += state[-network.n_reac-network.n_prod+i]
     return xs
 
+def get_rate_coefficients_SLS(network,T,P,method="mexp",neglect_high_energy_collisions=False,high_energy_rate_tol=0.01):
+    """
+    Generate phenomenological rate coefficients using the simulation least squares method
+    """
```

Comment:
Maybe add all available methods in the docstring

Reply:
added